From 6ae8b2b3140475b1a70485052454210aba4065a6 Mon Sep 17 00:00:00 2001
From: Jes Sorensen <Jes.Sorensen@gmail.com>
Date: Wed, 5 Apr 2017 15:22:36 -0400
Subject: [RHEL7.5 PATCH 056/169] Grow: Stop bothering about md driver
 versions older than 0.90.00

Signed-off-by: Jes Sorensen <Jes.Sorensen@gmail.com>
---
 Grow.c | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/Grow.c b/Grow.c
index 78a3474..15f4ed1 100755
--- a/Grow.c
+++ b/Grow.c
@@ -288,16 +288,9 @@ int Grow_addbitmap(char *devname, int fd, struct context *c, struct shape *s)
 	struct supertype *st;
 	char *subarray = NULL;
 	int major = BITMAP_MAJOR_HI;
-	int vers = md_get_version(fd);
 	unsigned long long bitmapsize, array_size;
 	struct mdinfo *mdi;
 
-	if (vers < 9003) {
-		major = BITMAP_MAJOR_HOSTENDIAN;
-		pr_err("Warning - bitmaps created on this kernel are not portable\n"
-			"  between different architectures.  Consider upgrading the Linux kernel.\n");
-	}
-
 	/*
 	 * We only ever get called if s->bitmap_file is != NULL, so this check
 	 * is just here to quiet down static code checkers.
-- 
2.7.4

